recipes/_compute_base_config.rb

```diff
@@ -26,6 +26,31 @@
 
 nfs_master = node['cfncluster']['cfn_master']
 
+# EFS
+# Get shared_dir path and mount EFS filesystem
+efs_shared_dir = "/"+node['cfncluster']['cfn_efs_shared_dir'].split(',')[0]
+
+# Check to see if EFS is created
+if efs_shared_dir != "/NONE"
+    # Create the EFS shared directory
```

Comment:
Please use 2 (not 4) spaces for indentation. See [IndentationWidth](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/IndentationWidth).